modules/gdscript/tests/scripts/analyzer/errors/virtual_super_not_implemented.out

```diff
@@ -0,0 +1,2 @@
+GDTEST_ANALYZER_ERROR
+Cannot call virtual function "_init()".
```

Comment:
Would you check if this feels better now? :)

Reply:
Ok, but I'm worried about the confusion between `super()` and `super._init()`. Maybe something like `Cannot call parent class function "_init()" because it hasn't been defined.`?

Reply:
Yeah, given that there was already some confusion by other people on RocketChat I think that makes a lot more sense!